### ISSUE TITLE
Request: More detailed installation instructions or how to

### ISSUE THREAD

**Issue by jasonwtobe** [tedpearson/weather2influxdb#1]
Being fairly new to Linux I can't make sense of the installation instructions. If possible could you add some more detail to the instructions for those of us who aren't native in Linux? I'm looking to supplement my smarthome logging ([setup following this blog](https://simonhearne.com/2020/pi-influx-grafana/)) with local weather using weather2influxdb

**Comment by tedpearson** [comment]
Hi Jason, I'm not quite sure if the update I just made is any more readable for you. Let me know if there's anything in particular you're finding difficult and perhaps I can explain it more simply for you.

Note that I haven't packaged this up in this git repo to install nicely and automatically run on your (I assume) raspberry pi. For my personal pi, I have some ansible configuration that installs this in `/usr/local/bin`, puts the config in `/usr/local/etc`, and creates a systemd service that starts it on boot.

Leaving this open in the interest of communication.

**Comment by tedpearson** [comment]
Closing issue due to no response for 4 months after updated docs.
